cypress/integration/spec.js

```diff
@@ -34,75 +44,136 @@ describe('Chart Explorer', function() {
     cy.visit('/', visitOptions)
 
     // load example data
-    cy.get('[data-test=example-link]').click()
+    cy.get('[data-dw=example-link]').click()
 
     // set some chart options
     cy.get(
-      ':nth-child(1) > [data-test=encoding-bar] > .css-1y3q4ke > .react-select__control > .react-select__indicators > .react-select__indicator'
+      '[data-dw=encoding-container-0] > [data-dw=encoding-bar] > .css-1y3q4ke > .react-select__control > .react-select__indicators > .react-select__indicator'
```

Comment:
this `.css-1y3q4ke` is interesting... I see it was already here. Can you explain to me where this class is coming from?

Reply:
I think it's some dynamic css class that gets created somehow, but I actually removed it because it's not needed since we're already being specific enough